src/main/java/org/apache/accumulo/testing/continuous/BulkIngest.java

```diff
@@ -50,13 +51,14 @@
   @Override
   public int run(String[] args) throws Exception {
     String ingestInstanceId = UUID.randomUUID().toString();
+    String bulkPath = args[0];
 
     Job job = Job.getInstance(getConf());
     job.setJobName("BulkIngest_" + ingestInstanceId);
     job.setJarByClass(BulkIngest.class);
     // very important to prevent guava conflicts
     job.getConfiguration().set("mapreduce.job.classloader", "true");
-    FileSystem fs = FileSystem.get(job.getConfiguration());
+    FileSystem fs = FileSystem.get(URI.create(bulkPath), job.getConfiguration());
```

Comment:
Do you know if paths w/o a scheme like `/tmp/bt/1` still work?

Reply:
@keith-turner , thanks for reviewing. Yes, it works w/o scheme, with (default or otherwise) scheme as well